### PR TITLE
ILogger - Exception.Message is populated as ExceptionTelemetry.Message, Bump version

### PR DESCRIPTION
**Pull request by cijothomas**
ILogger - If an exception is passed to log, then Exception.Message is populated as ExceptionTelemetry.Message. 

If TrackExceptionsAsExceptionTelemetry is false, then Exception.Message is stored as custom property "ExceptionMessage"

Foe example, consider the following log call.
testLogger.LogError(new Exception("ExceptionMessage"), "LoggerMessage");
The ExceptionTelemetry created will contain "ExceptionMessage" in its Message field. "LoggerMessage" will be in custom property "FormattedMessage"

If TrackExceptionsAsExceptionTelemetry is false, then TraceTelemetry is created, and "LoggerMessage" will be in its Message field. "ExceptionMessage" will be in custom property "ExceptionMessage"

Prior to this, when an exception is passed to log, its Exception.Message was not stored anywhere.

Also bumped and updated versions.

### CODE REVIEW

**Comment by RamjotSingh** [comment]
How does this behave when users use LogError with exception object in it? I think we might be regressing that here.

**Comment by cijothomas** [comment]
@RamjotSingh i have edited to give example. (I submitted PR with empty description, sorry)

File: src/ILogger/ApplicationInsightsLogger.cs [hunk 1]
```diff
@@ -86,16 +86,22 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
...
+                    if (exception != null)
+                    {
+                        traceTelemetry.Properties.Add("ExceptionMessage", exception.Message);
+                    }
+
                     this.telemetryClient.TrackTrace(traceTelemetry);
                 }
                 else
                 {
                     ExceptionTelemetry exceptionTelemetry = new ExceptionTelemetry(exception)
```
Comment (lmolkova) [inline src/ILogger/ApplicationInsightsLogger.cs]: this branch seems to be executed when exception == null  && this.applicationInsightsLoggerOptions.TrackExceptionsAsExceptionTelemetry (else), but it does not seem right to track exception and  Message = exception.Message will throw. Am I missing something?

Comment (cijothomas) [reply src/ILogger/ApplicationInsightsLogger.cs]: discussed offline. this is not the case.

**Pull request merged**